hawkular-inventory-dist/pom.xml

```diff
@@ -138,47 +138,20 @@
 
     <dependency>
       <groupId>org.umlg</groupId>
-      <artifactId>sqlg-hsqldb</artifactId>
+      <artifactId>sqlg-hsqldb-dialect</artifactId>
       <version>${version.org.umlg}</version>
-      <exclusions>
-        <exclusion>
-          <groupId>org.apache.tinkerpop</groupId>
-          <artifactId>gremlin-groovy</artifactId>
-        </exclusion>
-      </exclusions>
     </dependency>
 
     <dependency>
       <groupId>org.umlg</groupId>
-      <artifactId>sqlg-postgres</artifactId>
+      <artifactId>sqlg-postgres-dialect</artifactId>
       <version>${version.org.umlg}</version>
-      <exclusions>
-        <exclusion>
-          <groupId>org.apache.tinkerpop</groupId>
-          <artifactId>gremlin-groovy</artifactId>
-        </exclusion>
-      </exclusions>
     </dependency>
 
     <dependency>
       <groupId>org.umlg</groupId>
-      <artifactId>sqlg-h2</artifactId>
+      <artifactId>sqlg-h2-dialect</artifactId>
```

Comment:
Yeah, we probably should. I chose HSQLDB as the default simply because it's more stable in Sqlg but on the other hand, H2 is included in Wfly by default so we would not have to ship the drivers for it. I need to do more testing with H2 backend to see if we could switch to it.